packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-vdsm/tasks/libvirt_presets_configure.yml

```diff
@@ -0,0 +1,18 @@
+- block:
+  - name: create custom libvirt preset
+    copy:
+      dest: /usr/lib/systemd/system-preset/88-ovirt-monolithic-libvirt.preset
+      remote_src: yes
```

Comment:
I don't think `remote_src` is needed if you use `content`

Reply:
ack